packages/box/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "es6",
+    "noImplicitAny": true,
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "outDir": "dist",
+    "baseUrl": ".",
+    "lib": ["es2017"],
+    "paths": {
+      "web3": [
+        "../../node_modules/@types/web3/index",
+        "node_modules/web3/index"
+      ]
+    },
+    "rootDir": ".",
+    "typeRoots": [
+      "./typings",
+      "node_modules/@types",
```

Comment:
(I still think we should list the ones we use individually)

Reply:
What was the reasoning for this again?

Using the monorepo, this should direct appropriately, using the pkg individually, this should also direct appropriately.

Reply:
I ran into a problem where `@types` dependencies for package A were breaking package B. Specifically, things like `lib` being different in different tsconfigs (like maybe package A specifies `dom.d.ts`, or somesuch)

Reply:
In fact, previously, the types were included wholesale like that, and I changed it to be explicit. Here's the PR https://github.com/trufflesuite/truffle/pull/1965

Reply:
Ah, this is probably a good reason to set things up using the Quramy example.